src/utils/PatchError.js

```diff
@@ -52,8 +52,8 @@ export default class PatchError extends Error {
     for (let line = displayStartLine; line <= displayEndLine; line++) {
       let startOfLine = lineMap.indexForLocation({ line, column: 0 });
       let endOfLine = lineMap.indexForLocation({ line: line + 1, column: 0 });
-      if (isNaN(endOfLine)) {
-        if (isNaN(startOfLine)) {
+      if (endOfLine == null) {
```

Comment:
:thumbsup: yeah, I noticed that flow checking wasn't enabled, just calling that out as maybe something to think about. In a flow project I worked on recently, I ended up deciding to always use `== null` and `!= null` when working with `?` types.

I'm not sure where it would be in the docs (if anywhere), but here's an example:
https://tryflow.org/?code=bGV0IGZvbzogP251bWJlciA9IG51bGw7CmxldCBiYXI6IG51bWJlciA9IGZvbyA9PT0gbnVsbCA/IDAgOiBmb287CmxldCBiYXo6IG51bWJlciA9IGZvbyA9PSBudWxsID8gMCA6IGZvbzs=

The second line doesn't typecheck because flow thinks that a `?number` could still be `undefined` after a `=== null` check.